### DISCUSSION TITLE
Simple Testing Site

### DISCUSSION THREAD

**Discussion by timeshift92** [OrchardCMS/OrchardCore#9597] [Q&A]
Hello, I decided to create a simple test site,
what I could create on the bare .net core is very complicated, can anyone look at the source and say what I'm doing wrong?

https://github.com/timeshift92/OrchardCore

**Comment by jptissot** [comment]
The link you posted is not accessible.

**Reply by timeshift92** [reply]
My bad, i changed visibility

**Reply by Piedone** [reply]
What is the issue you need help with, exactly?

**Reply by timeshift92** [reply]
If you try to run the project, you can see that I created the content of the page inside it there are directions and inside the directions there are questions and inside the questions there are answers, and if this is all so the page starts loading terribly slow

**Reply by timeshift92** [reply]
I don't know if I can create something like pagination and lazy loading there, if so, please direct me

**Reply by Piedone** [reply]
It would help if your solution could be built :). Try to clone it anew and fix if you see errors. Lombiq.HelpfulExtensions is not loading. Also, add a NuGet.config like this one so people can easily use the preview packages: https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/blob/dev/NuGet.config

Otherwise, I suggest using Mini Profiler first to see what slows things down: https://docs.orchardcore.net/en/dev/docs/reference/modules/MiniProfiler/
